modules/llm-cache/radix-tree/radix-tree.cc

```diff
@@ -490,12 +490,10 @@ std::shared_ptr<RadixTree> RadixTree::Deserialize(std::string data) {
 
     raxNode* node = nullptr;
     VLOG(100) << "stage 1";
-    VINEYARD_ASSERT(radixTree->tree != nullptr);
 
-    // TBD refator this code.
+    // TBD refactor this code.
     node = raxFindAndReturnDataNode(radixTree->tree, subTreeTokenList[i].data(),
                                     subTreeTokenList[i].size(), NULL, false);
-    VINEYARD_ASSERT(node != nullptr);
```

Comment:
Use `RETURN_ON_ASSERT(..)` if you need to check some condtions.

Reply:
OK
Now I use RETURN_ON_ASSERT and RETURN_ON_ERROR to check condition.